policyengine_us/variables/gov/states/mt/tax/income/mt_taxable_income_indiv.py

```diff
@@ -15,47 +15,14 @@ class mt_taxable_income_indiv(Variable):
 
     def formula(person, period, parameters):
         mt_agi = person("mt_agi", period)
-        standard_deduction = add(
-            person.tax_unit, period, ["mt_standard_deduction_indiv"]
+        deductions_and_exemptions = person.tax_unit(
+            "mt_tax_unit_deductions_exemptions_indiv", period
         )
-        itemized_deductions = add(
-            person.tax_unit, period, ["mt_itemized_deductions_indiv"]
-        )
-        # Tax units can claim the larger of the itemized or standard deductions
-        itemizes = person.tax_unit("mt_tax_unit_itemizes", period)
-        deductions = where(itemizes, itemized_deductions, standard_deduction)
-        exemptions = add(person.tax_unit, period, ["mt_exemptions_indiv"])
-        deductions_and_exemptions = deductions + exemptions
-        # Allocate the exemptions and deductions based on the following process:
-        # 1. Determine whether the head or spouse AGI is larger
         head = person("is_tax_unit_head", period)
-        spouse = person("is_tax_unit_spouse", period)
-        head_agi = mt_agi * head
-        spouse_agi = mt_agi * spouse
-        spouse_agi_attributed_to_head = person.tax_unit.sum(spouse_agi) * head
-        head_over_spouse_agi = head_agi > spouse_agi_attributed_to_head
-        # 2. Determine the difference between the two AGIs and cap the deductions at that amount
-        difference = person.tax_unit.sum(
-            np.abs(head_agi - spouse_agi_attributed_to_head)
-        )
-        deductions_capped_at_agi_difference = min_(
-            difference, deductions_and_exemptions
-        )
-        # 3. Halve the remaining deductions amount
-        halved_capped_deductions = (
-            deductions_and_exemptions - deductions_capped_at_agi_difference
-        ) * 0.5
-        halved_deductions_allocated_to_head_or_spouse = (
-            halved_capped_deductions * (head | spouse)
-        )
-        # 4. Allocate the difference to the higher AGI and the halved deductions to each head and spouse
-        difference_applied_to_larger_agi = (
-            deductions_capped_at_agi_difference
-            * where(head_over_spouse_agi, head, spouse)
-        )
-        reduced_agi = (
-            mt_agi
-            - difference_applied_to_larger_agi
-            - halved_deductions_allocated_to_head_or_spouse
+        head_deductions = person.tax_unit(
+            "mt_head_deductions_exemptions_indiv", period
         )
-        return max_(reduced_agi, 0)
+        spouse_deductions = deductions_and_exemptions - head_deductions
+        capped_head_agi = max_(mt_agi - head_deductions, 0)
```

Comment:
Just a note that I think this approach correctly handles cases where taxable income for one spouse gets capped at zero, since it will appropriately move more to the other.

Reply:
I believe in cases where the total deductions and exemptions exceed the amount of AGI of both people the taxable income will be below 0 if this is not in place.
See test number 5 in the `mt_taxable_income_indiv.yaml` which fails without the max_